actions/testmo-run-submit-thread/action.yml

```diff
@@ -39,15 +43,19 @@ runs:
         fi
         ## submit results?
         SUCCESS=0
-        if [ ${REPORT} -eq 1 ]; then
+        if [[ ${REPORT} -eq 1 ]]; then
           echo "submitting results to TESTMO run ..."
           ## not checking testmo_url and token as this should be
           ## called between "create" and "complete"
-          npx testmo automation:run:submit-thread \
```

Comment:
As Dan mentioned, it seems like the only real change that would have been needed in this part of the action is to include the `--thread-resources` flag and value – that should have already been built prior to this action running by the job/workflow calling your new action prior to this one. Rather than trying to build CLI flags guard against no resources file being passed, it would be simpler to simply create an empty dummy resources file if one doesn't exist (that's what I have done in other projects): `cat {} > resources-file.json` is a minimal “empty” file that Testmo accepts.

Reply:
cool, I didn't expect TestMo would accept an empty resources file.

Reply:
Yup, I did so here in a separate version of these actions for test-automation:

https://github.com/neuralmagic/test-automation/blob/main/.github/actions/testmo-run-submit-thread/action.yml#L40-L43

As you can see, it's a much simpler bit of logic that eliminates the need for more complex logic later on.